CodeGeneration/Sources/SyntaxSupport/gyb_generated/AvailabilityNodes.swift

```diff
@@ -81,14 +66,11 @@ public let AVAILABILITY_NODES: [Node] = [
        kind: "Syntax",
        children: [
          Child(name: "Platform",
-               kind: "IdentifierToken",
+               kind: .token(choices: [.token(tokenKind: "IdentifierToken")]),
                description: "The name of the OS on which the availability should berestricted or 'swift' if the availability should berestricted based on a Swift version.",
```

Comment:
I'm not actually sure what these are used for. But it appears we're not adding a space when we drop the newlines and indentation (separate to this PR, just noticed it here).

Reply:
What exactly do you mean?